src/dotnet/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager" Version="1.4.0" />
+    <PackageReference Include="Jint" Version="2.11.58" />
+    <PackageReference Include="LibGit2Sharp" Version="0.27.2" />
+    <PackageReference Include="Markdig" Version="0.31.0" />
```

Comment:
Can you share how these references were chosen and how they were approved for use?

Reply:
Any process I can follow to use these external nuget packages?

Reply:
I don't know where the official process lives these days, but we'll need to make sure that the Microsoft Open Source office has signed off on using these to be sure there are no concerns with licenses.

Reply:
3 nuget packages referenced for handling markdown, yaml and git operation:
 - For yaml and git, not corresponding microsoft library found and they are one of the most popular nuget package to use (recommended by Bing chat). 
 - For markdown, it's recommended by Microsoft Learning Site. https://learn.microsoft.com/en-us/windows/communitytoolkit/parsers/markdownparser

According to https://docs.opensource.microsoft.com/using/ ,  all the open sources used need to be registered in Component Governance (CG). And the registration usually will be handled automatically for repositories that have AzureDevOps pipeline. Could you help to confirm whether the pipeline for azure-sdk-tools repo has included the registration? or I need to do something to add a task in my pipeline to do the registration? thanks.

Reply:
I do not know; this is a question for the EngSys team.